tests/integration/test_bundle.py

```diff
@@ -29,31 +29,29 @@
 async def test_deploy_bundle(ops_test: OpsTest) -> None:
     """Deploy bundle."""
     async with ops_test.fast_forward():
+        logger.info("Deploy MySQL K8s bundle and wait for up and running")
         await ops_test.model.deploy("./releases/latest/mysql-k8s-bundle.yaml", trust=True)
-
-        await ops_test.model.wait_for_idle(apps=[MYSQL_APP], status="active", timeout=15 * 60)
-
         await ops_test.model.wait_for_idle(apps=[ROUTER_APP], status="waiting", timeout=5 * 60)
+        await ops_test.model.wait_for_idle(apps=[MYSQL_APP], status="active", timeout=15 * 60)
```

Comment:
the timeout for the mysql app was and is 15 mins (with an additional 5 mins due to the timeout on the wait for idle above). did you wan to increase the timeout (as per your PR description)?

Reply:
It was 5 minutes till [today](https://github.com/canonical/mysql-k8s-bundle/pull/29/files#diff-6292111ee427b839680db4b3caf53e52e71b52609c436242b86744d44c365ab1).
I have reordered it here as mysql-router starts instantly... so it is couple of seconds to not wait it later after mysql-cluster, Sure I can restore order here.